backend/src/aggregator.rs

```diff
@@ -102,6 +102,26 @@ impl Message for Subscribe {
     type Result = bool;
 }
 
+/// Message sent from the FeedConnector to the Aggregator consensus requested
+pub struct SendFinality {
+    pub chain: Label,
+    pub fid: FeedId,
+}
+
+impl Message for SendFinality {
```

Comment:
Don't need the result anymore, so can just `#[derive(Message)]` on top of the `SendFinality` struct.